receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -30,81 +36,206 @@ metrics:
     gauge:
       value_type: int 
     attributes: [splunk.index.name]
+  splunk.scheduler.avg.execution.latency:
+    enabled: true
+    description: Gauge tracking the average execution latency of scheduled searches
+    unit: '{ms}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.scheduler.completion.ratio:
+    enabled: true
+    description: Gauge tracking the ratio of completed to skipped scheduled searches
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.indexer.avg.rate:
+    enabled: true
+    description: Gauge tracking the average rate of indexed data. **Note:** Search is best run against a Cluster Manager.
+    unit: KBy
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.pipeline.set.count:
+    enabled: true
+    description: Gauge tracking the number of pipeline sets per indexer. **Note:** Search is best run against a Cluster Manager.
+    unit: KBy
+    gauge:
+      value_type: int 
+    attributes: [splunk.host]
+  splunk.parse.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer parser queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.aggregation.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer aggregation queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]  
+  splunk.typing.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer typing queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]  
+  splunk.indexer.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer index queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.scheduler.average.run.time:
+    enabled: true
+    description: Gauge tracking the average runtime of scheduled searches
+    unit: '{ms}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.indexer.raw.write.seconds:
```

Comment:
I think we don't put seconds in metric names usually

Reply:
Thanks for the heads up! Didn't know this doc existed but probably should have :D 
https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#units